hnn_core/tests/test_extracellular.py

```diff
@@ -210,15 +210,13 @@ def test_rec_array_calculation():
     hnn_core_root = op.dirname(hnn_core.__file__)
     params_fname = op.join(hnn_core_root, 'param', 'default.json')
     params = read_params(params_fname)
-    params.update({'t_evprox_1': 7,
-                   't_evdist_1': 17})
+    params.update({'t_evprox_1': 7, 't_evdist_1': 17})
     net = jones_2009_model(params, mesh_shape=(3, 3),
                            add_drives_from_params=True)
 
     # one electrode inside, one above the active elements of the network,
     # and two more to allow calculation of CSD (2nd spatial derivative)
-    electrode_pos = [(1, 2, 1000), (2, 3, 3000), (3, 4, 5000),
-                     (4, 5, 7000)]
+    electrode_pos = [(1, 2, 1000), (2, 3, 3000), (3, 4, 5000), (4, 5, 7000)]
```

Comment:
as a general rule, one should keep lines not relevant to the PR untouched. Formatting changes can be delegated to a PR of its own. The smaller your diff, the less comments, and easier it is for us to merge.

Reply:
Sorry, I think this happened because I had a flake8 error for this line (empty whitespace)

Reply:
Yes, I've noticed that there are many flake8 formatting errors in the codebase that were probably merged before flake8 was implemented as a gh action check,  this is because the gh action only checks edited files in the current PR, not the rest of the codebase. So these error are only flagged when a contributor edits a file with one of these legacy errors. I can make separate PR that sweeps the codebase for these legacy formatting errors.